docs/rest-api/source/includes/wp-api-v3/deposits.md

```diff
@@ -0,0 +1,575 @@
+			{
+				"amount": -114696,
+				"currency": "eur",
+				"source_types": {
+					"card": -114696
+				},
+				"deposits_count": 1
+			},
+			{
+				"amount": 707676,
+				"currency": "usd",
+				"source_types": {
+					"card": 707676
+				},
+				"deposits_count": 2
+			}
+		],
+		"available": [
+			{
+				"amount": 573480,
+				"currency": "eur",
+				"source_types": {
+					"card": 573480
+				}
+			},
+			{
+				"amount": 587897,
+				"currency": "usd",
+				"source_types": {
+					"card": 587897
+				}
+			}
+		],
+		"instant": [
+			{
+				"amount": 12345,
+				"currency": "usd",
+				"fee": 185,
+				"fee_percentage": 1.5,
+				"net": 0,
+				"transaction_ids": [
+					"txn_3OHyIxCIHGKp1UAi0aVyDQ5D",
+					"txn_3OJSuOCIHGKp1UAi1mRA2lL5"
+				]
+			}
+		]
+	},
+	"account": {
+		"deposits_enabled": true,
+		"deposits_blocked": false,
+		"deposits_schedule": {
+			"delay_days": 7,
+			"interval": "weekly",
+			"weekly_anchor": "friday"
+		},
+		"default_currency": "eur"
+	}
+}
+```
+
+## Get deposits overview for single account deposit currency
+
+Fetch an overview of account deposits for a single deposit currency. This includes details for the last paid deposit, next scheduled deposit, and last manual deposits.
+
+### HTTP request
+
+<div class="api-endpoint">
+	<div class="endpoint-data">
+		<i class="label label-get">GET</i>
+		<h6>/wp-json/wc/v3/payments/deposits/overview</h6>
+	</div>
+</div>
+
+### Returns
+
+-   `last_deposit` _object_ | _null_- The last deposit that has been paid for the deposit currency.
+-   `next_deposit` _object_ | _null_ - The next scheduled deposit for the deposit currency.
+-   `balance` _object_
+    -   `pending` _object_ - The pending balance for the deposit currency.
+        -   `amount` _int_ - The amount of the balance.
+        -   `currency` _string_ - The currency of the balance. E.g. `usd`.
+        -   `source_types` _object_ - The amount of the balance from each source type, e.g. `card` or `financing`.
+        -   `deposits_count` _int_ - The number of deposits that make up the balance.
+    -   `available` _object_ - The available balance for the deposit currency.
+        -   `amount` _int_ - The amount of the balance.
+        -   `currency` _string_ - The currency of the balance. E.g. `usd`.
+        -   `source_types` _object_ - The amount of the balance from each source type, e.g. `card` or `financing`.
+-   `instant_balance` _object_ | _null_ - The instant balance for the deposit currency.
+    -   `amount` _int_ - The amount of the balance.
+    -   `currency` _string_ - The currency of the balance. E.g. `usd`.
+    -   `fee` _int_ - The fee amount of the balance.
+    -   `fee_percentage` _int_ - The fee percentage of the balance.
+    -   `net` _int_ - The net amount of the balance.
+    -   `transaction_ids` _array_ - The list of transaction IDs that make up the balance.
+-   `account` _object_
+    -   `deposits_disabled` _bool_ - Whether deposits are enabled for the account.
```

Comment:
After reading this, it becomes apparent that this is not consistent with /overview-all API where over there, we use `deposits_enabled` instead of `deposits_disabled`. Just an observation.

Reply:
Also, I don't think `/deposits/overview` is used at all in client, unsure about mobile– TBC.

We can probably deprecate this endpoint entirely and remove it later.